docs/UserGuide.md

```diff
@@ -264,12 +301,14 @@ Coming soon!
 
 |Feature|Command|
 |---|---|
-| Help               | `help`|
-| add user           | `adduser -n <USERNAME> -dob <dd/MM/yyyy> -g <GENDER>` |
-| Switch user        | `switchuser -n <USERNAME>` |
-| Browse             | `browse -s [name/rating] -p <1-26> -o [asc/dsc]` |
-| Create watchlist   | `watchlist -n <WATCHLIST_NAME>` |
-| List all watchlist | `watchlist -l` |
-| Select watchlist   | `watchlist -s <WATCHLIST_INDEX>` |
-| Delete watchlist   | `watchlist -d <WATCHLIST_INDEX>` |
-| Add to watchlist   | `add -a <ANIME_NAME>` |
\ No newline at end of file
+| Help               | `help`                                                   |
+| add user           | `adduser -n <USERNAME> -dob <dd/MM/yyyy> -g <GENDER>`    |
```

Comment:
Capitalize the first letter, same for line 314.

Reply:
Got it! I have rectified the issue. Thank you